addon/model/model-node.ts

```diff
@@ -156,6 +213,9 @@ export default abstract class ModelNode {
   setTextAttribute(_key: TextAttribute, _value: boolean) {
   }
 
+  /**
+   * @deprecated use {@link ModelTreeWalker} instead
```

Comment:
perhaps (though I guess that would make the PR explode) we can just remove and replace with modeltreewalker where it's used

Reply:
yeah too much cognitive load for this pr, but we should do it soon